modules/backend/formwidgets/sensitive/assets/js/sensitive.js

```diff
@@ -0,0 +1,61 @@
++function ($) { "use strict";
+
+    var Sensitive = function(element, options) {
+        this.$el = $(element)
+        this.init()
+    }
+
+    Sensitive.DEFAULTS = {
+        readOnly: false,
+        disabled: false
+    }
+
+    Sensitive.prototype.init = function() {
+        var self = this
+        this.$sensitiveField = this.$el
+        this.$showIcon = this.$el.siblings('.toggle-icon:first')
+
+        this.$showIcon.click(function(){
+            self.togglePassword()
+            self.toggleIcon()
+        })
+    }
+
+    Sensitive.prototype.togglePassword = function() {
+        if (this.$sensitiveField.attr("type") == "password") {
+            this.$sensitiveField.attr("type", "text");
+        } else {
+            this.$sensitiveField.attr("type", "password");
```

Comment:
We prefer strings to be defined with single quotes.

```suggestion
        if (this.$sensitiveField.attr('type') === 'password') {
            this.$sensitiveField.attr('type', 'text');
        } else {
            this.$sensitiveField.attr('type', 'password');
```